spec/controllers/completions_cancellation_controller_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe CompletionsCancellationController do
+  let(:user) { build_stubbed(:user, :fully_registered) }
+
+  before do
+    stub_sign_in(user)
+  end
+
+  it 'renders the show template' do
+    stub_analytics
+
+    expect(@analytics).to receive(:track_event).with(
+      :exit_to_sp_confirmation_page_visited,
+    )
+
+    get :show
```

Comment:
I'd recommend using `have_logged_event` over stubbing `track_event`, since it includes a bunch of additional helpers from [`FakeAnalytics`](https://github.com/18F/identity-idp/blob/main/spec/support/fake_analytics.rb) (PII spill detector, undocumented arguments checker, etc).

```suggestion
    get :show
    
    expect(@analytics).to have_logged_event(:completions_cancellation_visited)
```